### PR TITLE
Onboarding

### PR DESCRIPTION
**Pull request by jleveugle**

### CODE REVIEW

File: packages/manager/apps/public-cloud/src/index.html [hunk 1]
```diff
@@ -25,6 +25,9 @@
                     </div>
                 </div>
             </div>
+            <a data-ng-if="$ctrl.feedbackUrl" href="{{ $ctrl.feedbackUrl }}" target="blank" style="position: fixed; right: 2rem; bottom: 2rem;">
```
Comment (antleblanc) [inline packages/manager/apps/public-cloud/src/index.html]: Consider adding `rel="noopener"` would be great.

```suggestion
            <a data-ng-if="$ctrl.feedbackUrl" href="{{ $ctrl.feedbackUrl }}" target="_blank" style="position: fixed; right: 2rem; bottom: 2rem;">
```

File: packages/manager/apps/public-cloud/src/index.html [hunk 2]
```diff
@@ -25,6 +25,9 @@
...
+                <img alt="Feedback" data-ng-src="{{ $ctrl.feedback }}" />
```
Comment (antleblanc) [inline packages/manager/apps/public-cloud/src/index.html]: No translation needed (alt attribut I mean)?

File: packages/manager/modules/pci/src/projects/onboarding/index.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import './onboarding.less';
+
+const moduleName = 'ovhManagerPciProjectsOnBoardingLazyLoading';
+
+angular
+  .module(moduleName, [
+    'ui.router',
+    'oc.lazyLoad',
+  ])
+  .config(($stateProvider) => {
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/onboarding/index.js]:
```suggestion
  .config(/* @ngInject */ ($stateProvider) => {
```

File: packages/manager/modules/pci/src/projects/onboarding/onboarding.html [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<div class="pci_projects_onboarding">
+    <div class="row">
+        <div class="col-md-6">
+            <h1 data-translate="pci_projects_onboarding_title"></h1>
+            <p data-translate="pci_projects_onboarding_description1"></p>
+            <p data-translate="pci_projects_onboarding_description2"></p>
+            <p data-translate="pci_projects_onboarding_description3"></p>
+            <oui-button class="mt-4" variant="primary">
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/onboarding/onboarding.html]: `data` prefix is used everywhere into this template file. I encourage you to stick with it.

```suggestion
            <oui-button class="mt-4" data-variant="primary">
```

File: packages/manager/modules/pci/src/projects/onboarding/onboarding.html [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+                <span data-translate="pci_projects_onboarding_create_project"></span>
+            </oui-button>
+        </div>
+    </div>
+    <img data-ng-src="{{ $ctrl.illustration }}" alt="" />
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/onboarding/onboarding.html]: No extra self-closing tag.

```suggestion
    <img data-ng-src="{{ $ctrl.illustration }}" alt="">
```

File: packages/manager/apps/public-cloud/package.json [hunk 6]
```diff
@@ -67,6 +67,7 @@
   },
   "devDependencies": {
     "@ovh-ux/manager-webpack-config": "^3.0.6",
+    "webpack": "^4.30.0",
```
Comment (marie-j) [inline packages/manager/apps/public-cloud/package.json]: Isn't this brought with `manager-webpack-config` ? https://github.com/ovh-ux/manager-webpack-config/blob/master/package.json#L60

File: packages/manager/apps/public-cloud/src/index.html (hunk 1 above)
Comment (marie-j) [inline packages/manager/apps/public-cloud/src/index.html]:
```suggestion
            <a data-ng-if="$ctrl.feedbackUrl" data-ng-href="{{ $ctrl.feedbackUrl }}" target="blank" style="position: fixed; right: 2rem; bottom: 2rem;" rel="noopener">
```

File: packages/manager/modules/pci/src/projects/onboarding/onboarding.controller.js [hunk 7]
```diff
@@ -0,0 +1,8 @@
+import illustration from './onboarding.png';
+
+export default class {
+  /* @ngInject */
```
Comment (marie-j) [inline packages/manager/modules/pci/src/projects/onboarding/onboarding.controller.js]: Not really needed ?
```suggestion

```

**Pull request merged**